src/index.js

```diff
@@ -0,0 +1,34 @@
+// import _ from 'lodash';
+
+import './style.css';
+
+const listTable = [{
+  description: 'go to the market',
+  completed: true,
+  index: 0,
+},
+{
+  description: 'wash dishes',
+  completed: true,
+  index: 1,
+},
+{
+  description: 'play guitar',
+  completed: true,
+  index: 2,
+},
+'rwanda',
+];
+
+const todoList = document.querySelector('.todo-list');
+// eslint-disable-no-use-before-define
```

Comment:
- [ ] Remove this line because and I know it can get frustrating but linters are there to help you write a better code.